### PR TITLE
#1 - Backend: Spring boot base project

### PR DESCRIPTION
**Pull request by olegbal**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	repositories {
+		mavenCentral()
+	}
+	dependencies {
+		classpath("org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}")
+	}
+}
+
+apply plugin: 'java'
+apply plugin: 'eclipse'
```
Comment (vlsidlyarevich) [inline build.gradle]: if you added eclipse plugin, why not added for idea?

File: build.gradle [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+apply plugin: 'org.springframework.boot'
+
+jar{
+
```
Comment (vlsidlyarevich) [inline build.gradle]: remove empty spaces

File: build.gradle [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+sourceCompatibility = 1.8
+
+repositories {
+	mavenCentral()
+}
+
+
+dependencies {
+	compile('org.springframework.boot:spring-boot-starter')
+    compile 'org.springframework.boot:spring-boot-starter-web'
```
Comment (vlsidlyarevich) [inline build.gradle]: please, use code formatting, and use one common style (make brackets or not)

File: src/main/java/com/github/dregheap/DregHeapBackendApplication.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
+package com.github.dregheap;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class DregHeapBackendApplication {
```
Comment (vlsidlyarevich) [inline src/main/java/com/github/dregheap/DregHeapBackendApplication.java]: Don't use words like 'backend', 'class' and so on in class namings. And don't use word 'application'. Just 'DregHeap' will be enough.

**Review by vlsidlyarevich — APPROVED** [review]
It's ok

**Comment by solairerove** [comment]
the first нах

**Pull request merged**